tasks/eulers-sum-of-powers-conjecture/src/main.rs

```diff
@@ -1,7 +1,7 @@
-const MAX_N: u64 = 250;
+const MAX_N: usize = 250;
```

Comment:
Why does this type need to be changed?

Reply:
It's a nit, but `u64`->`usize` casting gives you a warning (for 32-bit platforms), while `usize`->`u64` casting does not give a warning.

Reply:
I see. I'm not super happy with all the `as`-casting going on in the rest of this solution, but that doesn't need to be addressed here.